module/certificateTemplates/helper.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * name : helper.js
+ * author : Vishnu
+ * created-date : 29-sep-2022
+ * Description : Certificate Template related helper functionality.
+*/
+// dependencies
+const filesHelpers = require(ROOT_PATH+"/module/cloud-services/files/helper");
+const request = require("request");
+
+const { ObjectId } = require("mongodb");
+
+/**
+    * CertificateTemplatesHelper
+    * @class
+*/
+module.exports = class CertificateTemplatesHelper {
+
+  /**
+   * Create certificate template.
+   * @method create
+   * @name create
+   * @param {Object} data - certificate template creation data.
+   * @returns {JSON} created certificate template details. 
+   */
+  
+  static create(data) {
+    return new Promise(async (resolve, reject) => {
+        try {
+            let certificateTemplateCreated = 
+            await database.models.certificateTemplates.create(
+              data
+            );
+            return resolve({
+                message : constants.apiResponses.CERTIFICATE_TEMPLATE_ADDED,
+                data : {
+                  id : certificateTemplateCreated._id
+                }
+              });
+            
+        } catch (error) {
+            return reject(error);
+        }
+    });
+  }
+
+  /**
+   * Update certificate template.
+   * @method update
+   * @name update
+   * @param {String} templateId - certificate template Id.
+   * @param {Object} data - certificate template updation data.
+   * @returns {JSON} Updated certificate template details. 
+  */
+  
+  static update(templateId, data) {
+    return new Promise(async (resolve, reject) => {
+        try {
+            //  If templateUrl value passed as empty string. 
```

Comment:
@VISHNUDAS-tunerlabs see if we can change this and add it to the validator.

Reply:
@aks30 this is done to avoid error if supporting team passes empty string for templateUrl, programId or solutionId( these values can't be empty).  while using express validator validator also check for key, so update data without these keys not passing validation.